gemma/run.cc

```diff
@@ -192,10 +211,13 @@ void ReplGemma(const ThreadingArgs& threading, const InferenceArgs& inference,
       prompt_size = prompt.size();
     }
 
-    if constexpr (kVerboseLogTokens) {
-      for (int i = 0; i < prompt_size; ++i) {
-        fprintf(stderr, "DDD TOKEN %3d: %6d\n", i, prompt[i]);
-      }
+    if (have_image) {
+      runtime_config.image_tokens = &image_tokens;
+      prompt_size = prompt.size();
+      // The end of the prefix for prefix-LM style attention in Paligemma.
+      prefix_end = prompt_size;
+      // We need to look at all the tokens for the prefix.
+      runtime_config.prefill_tbatch_size = prompt_size;
```

Comment:
Why this change? It is OK to have smaller batches than the prompt. For prompts > 512 or so, we want to break them up into multiple batches. I think we can leave prefill_tbatch_size unchanged, no?